mmseg/models/backbones/stdcnet.py

```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
The original repo: https://github.com/MichaelFan01/STDC-Seg

Reply:
Ok